### PR TITLE
Apply PEP-7 to `Python/import.c`

### PR DESCRIPTION
**Pull request by picnixz**

### CODE REVIEW

**Comment by picnixz** [comment]
The base branch went wrong !

**Review by StanFromIreland — APPROVED** [review]
LGTM

**Comment by picnixz** [comment]
I think this one should be applied at the end of my chain of PR. All other PRs can be merged sequentially but this one causes a conflict, so it's better to make it last.

**Comment by picnixz** [comment]
FTR, I need to update the tests: `Empty module name` -> `empty module name`

File: Python/import.c [hunk 1]
```diff
@@ -3872,41 +3875,38 @@ _PyImport_LoadLazyImportTstate(PyThreadState *tstate, PyObject *lazy_import)
...
+        }
+        else {
             Py_INCREF(lz->lz_attr);
             fromlist = lz->lz_attr;
         }
     }
 
     PyObject *globals = PyEval_GetGlobals();
 
-    if (PyMapping_GetOptionalItem(lz->lz_builtins, &_Py_ID(__import__), &import_func) < 0) {
```
Comment (DinoV) [inline Python/import.c:3908]: This open bracket should be on the same line?

File: Python/import.c [hunk 2]
```diff
@@ -4319,11 +4323,14 @@ register_lazy_on_parent(PyThreadState *tstate, PyObject *name, PyObject *builtin
...
                 if (!contains) {
-                    PyObject *lazy_module_attr = _PyLazyImport_New(builtins, parent, child);
+                    PyObject *lazy_module_attr = _PyLazyImport_New(
+                        builtins, parent, child
+                    );
                     if (lazy_module_attr == NULL) {
                         goto done;
                     }
-                    if (PyDict_SetItem(parent_dict, child, lazy_module_attr) < 0) {
+                    if (PyDict_SetItem(parent_dict, child, lazy_module_attr) < 0)
```
Comment (DinoV) [inline Python/import.c]: Also should be on the same line

**Review by DinoV — APPROVED** [review]
Just a couple of small nits but otherwise LGTM

**Comment by picnixz** [comment]
After checking a bit why the test actually failed, it's because the message was like this since 2010. So I won't change "Empty module" to "empty module" (I thought it was a new message).

File: Python/import.c (hunk 1 above)
Comment (picnixz) [reply Python/import.c:3908]: Well... strictly speaking, the entire call should be wrapped weirdly according to PEP-7 but we're not exceeding the line for too much so I'm applying the principle of "let's not be nitpick" here. I'm just making sure that the code doesn't exceed too much the 80 chars width (otherwise I really need to scroll and it's a bit annoying I think for readers)

**Comment by picnixz** [comment]
@DinoV Actually, maybe you prefer to hold that PR in case something else changes in the main branch? I can definitely update it when nothing more needs to be added so that I only need to reformat the code once. WDYT?

**Comment by Yhg1s** [comment]
Let's get this in now. If we have merge conflicts with these changes we'd have them without these changes as well, and if we leave this pending we'd have *two* chances of getting merge conflicts :)
